examples/example_helper.rb

```diff
@@ -63,4 +63,75 @@ def terminate
   end
 end
 
+class RemoteExecutorHelper
+  class << self
+
+    def run_observer
+      world = ExampleHelper.create_world do |config|
+        config.persistence_adapter = persistence_adapter
+        config.connector           = connector
+        config.executor            = false
+      end
+      run(world)
+    end
+
+    def run_server
+      world = ExampleHelper.create_world do |config|
+        config.persistence_adapter = persistence_adapter
+        config.connector           = connector
+      end
+      run(world)
+    end
+
+    def client
+      @client ||= ExampleHelper.create_world do |config|
```

Comment:
This "create world, assign persistence adapter and connector" pattern repeats a lot, also there's another `#client` a bit below https://github.com/Dynflow/dynflow/pull/293/files#diff-1d1e81c6ffa6ce36582fbb4c1e6c2e59R126